.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Motivation
+Add references to relevant tickets or a short description about what motivated you do it. (E.g JIRA: https://issues.jboss.org/browse/AEROGEAR-{} AND/OR ISSUE: https://github.com/aerogear/unifiedpush-admin-ui/issues{}) 
+
+## What
```

Comment:
Maybe we should change these three to two "Why" and "How". What tends to feel redundant with motivation/jira/etc.